src/mem.rs

```diff
@@ -91,18 +103,6 @@ pub enum Flush {
     /// block before the empty fixed code block.
     Partial = ffi::MZ_PARTIAL_FLUSH as isize,
 
-    /// A deflate block is completed and emitted, as for `Flush::Sync`, but the
-    /// output is not aligned on a byte boundary and up to seven vits of the
-    /// current block are held to be written as the next byte after the next
-    /// deflate block is completed.
-    ///
-    /// In this case the decompressor may not be provided enough bits at this
-    /// point in order to complete decompression of the data provided so far to
-    /// the compressor, it may need to wait for the next block to be emitted.
-    /// This is for advanced applications that need to control the emission of
-    /// deflate blocks.
-    Block = ffi::MZ_BLOCK as isize,
-
```

Comment:
@fstirlitz I believe it's because it's not actually used in the backend, but do you have a use case for it?